Sources/UI/Experiences/Models/Row.swift

```diff
@@ -108,7 +108,7 @@ extension Row: Decodable {
 }
 
 extension Row: AttributeRepresentable {
```

Comment:
`AttributeRepresentable` has been removed in this PR. You'll need to update this to match the approach you took in the `Beacon` and `Geofence` classes.